src/sc2laddercore/LegacyLadderConfig.cpp

```diff
@@ -0,0 +1,77 @@
+#include "LegacyLadderConfig.h"
+
+#include <iostream>
+#include <sstream>
+#include <fstream>  
+
+LegacyLadderConfig::LegacyLadderConfig(const std::string &InConfigFile)
+	:ConfigFileLocation(InConfigFile)
+{
+}
+
+
+
+bool LegacyLadderConfig::ParseConfig()
```

Comment:
I vote for removal of all the legacy code. There are two reasons behind:
a. We have version control history so can always revert the changes (especially if it would be in a single commit).
b. Current implementation of the cmake routine will grab all the source files, including the legacy code, which means that we have to support this code in the future.

If you still want to keep both implementations, then probably we should use a kind of virtual interface class in order to force both implementation use the same interface and implement it in some way. That will allow to easily switch to a different config in the future.

Reply:
I would also like to remove it as well, but I kept it in because it was easy enough, just in case we decided to support backwards compatibility.
If no-one else has any qualms, I'll just remove it.